### PR TITLE
Disable CDN communication

### PR DESCRIPTION
**Pull request by 7hommay**
Adds a stubbed NetworkManager returning fixed responses for manifest and app config

Fixes https://github.com/minvws/nl-covid19-notification-app-coordination-private/issues/32

### CODE REVIEW

**Review by skos001 — DISMISSED** [review]

**Review by skos001 — DISMISSED** [review]

File: Sources/ENCore/Common/NetworkManager/NetworkManagerBuilder.swift [hunk 1]
```diff
@@ -87,13 +87,17 @@ private final class NetworkManagerDependencyProvider: DependencyProvider<Network
...
 
-        let dependencyProvider = NetworkManagerDependencyProvider(dependency: dependency)
-
-        return NetworkManager(configurationProvider: dependencyProvider.dependency.networkConfigurationProvider,
-                              responseHandlerProvider: dependencyProvider.responseHandlerProvider,
-                              storageController: dependencyProvider.dependency.storageController,
-                              session: dependencyProvider.session,
-                              sessionDelegate: dependencyProvider.sessionDelegate as? URLSessionDelegateProtocol,
-                              urlResponseSaver: dependencyProvider.urlResponseSaver)
+//        Disable network traffic
```
Comment (hvisser) [inline Sources/ENCore/Common/NetworkManager/NetworkManagerBuilder.swift:90]: Would be better to just remove this vs commenting this out IMHO

**Review by hvisser — CHANGES_REQUESTED** [review]

**Comment by nicktencate** [comment]
@Rool can you take a look?

**Comment by Rool** [comment]
I agree with @hvisser. Remove the commented out networkmanager in the builder

**Comment by Rool** [comment]
And three of the tests are failing:

ENCoreTests.EndOfLifeViewControllerTests
  test_snapshot_endOfLifeViewController, failed - Snapshot does not match reference.
  /Users/runner/work/nl-covid19-notification-app-ios/nl-covid19-notification-app-ios/Sources/ENCoreTests/App/Features/EndOfLifeViewController/EndOfLifeViewControllerTests.swift:35
  ```
    func test_snapshot_endOfLifeViewController() {
        snapshots(matching: viewController)
    }
  ```

  test_snapshot_endOfLifeViewController, failed - Snapshot does not match reference.
  /Users/runner/work/nl-covid19-notification-app-ios/nl-covid19-notification-app-ios/Sources/ENCoreTests/App/Features/EndOfLifeViewController/EndOfLifeViewControllerTests.swift:35
  ```
    func test_snapshot_endOfLifeViewController() {
        snapshots(matching: viewController)
    }
  ```

  test_snapshot_endOfLifeViewController, failed - Snapshot does not match reference.
  /Users/runner/work/nl-covid19-notification-app-ios/nl-covid19-notification-app-ios/Sources/ENCoreTests/App/Features/EndOfLifeViewController/EndOfLifeViewControllerTests.swift:35
  ```
    func test_snapshot_endOfLifeViewController() {
        snapshots(matching: viewController)
    }
  ```

**Comment by hvisser** [comment]
@Rool can you fix this in absence of @7hommay ?

**Comment by Rool** [comment]
Will try to fix this. Give me an hour or so.

**Comment by Rool** [comment]
I ran into openSSL errors while running the tests. So I temporarily disabled the failing snapshot tests.

**Comment by Rool** [comment]
The same three snapshots keep failing on CI.

**Comment by hvisser** [comment]
@Rool I've noticed ;) Working on it, CI has different results vs running these locally 🥲

**Review by Rool — DISMISSED** [review]

**Review by hvisser — APPROVED** [review]

**Pull request merged**